src/makeCalculator.js

```diff
@@ -37,7 +37,40 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const obj = {
```

Comment:
better use full word 'object' to name variables

Reply:
What is obj?